definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -277,7 +277,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "create",
     "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "RU"
+    "CRUD": "CRU"
```

Comment:
are you sure?

Reply:
updated RBAC "CRU" can be added in the "-prod" file and the original left as "RU"

Reply:
Suggestion: RU as is can be moved to the prod file and the CRU added to the feature file.

Reply:
Why are you changing this?

Reply:
This is a temp change for development. it was highlighted in the description in a commit that will be undone. This is just for testing

Reply:
fair enough.
Please leave this conversation open so we don't forget to revert it before merging.

Reply:
You have to revert, don't you?

Reply:
removed